javascript/widgets/config_test.js

```diff
@@ -242,6 +242,30 @@ function testGetTosUrl() {
 }
 
 
+function testRequiredDisplayNameShouldBeTrueByDefault() {
```

Comment:
For test names use following format:
testRequiredDisplayName_case1
testRequiredDisplayName_case2
testRequiredDisplayName_case3